capybara-bootstrap-datepicker.gemspec

```diff
@@ -28,4 +28,6 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'phantomjs', '~> 2.1', '>= 2.1.1.0'
   gem.add_development_dependency 'poltergeist', '~> 1.18', '>= 1.18.1'
   gem.add_development_dependency 'rspec', '~> 3.9', '>= 3.9.0'
+  gem.add_development_dependency 'webdrivers'
+  gem.add_development_dependency 'selenium-webdriver'
```

Comment:
I am not used to update gemspec file, so that I have not marked versions.
I would require some help for that.

Reply:
The syntax is the same as in a Gemfile. The site [rubygems.org](https://rubygems.org/) provides an example of the correct syntax based on the latest version available on the detail page of each gem. For example :

```ruby
gem 'webdrivers', '~> 4.6'
```

Just convert it to the Gemspec syntax:

```ruby
gem.add_development_dependency 'webdrivers' , '~> 4.6'
```

Reply:
Also, it seems that `selenium-webdriver` is a dependency of `webdrivers`, so you may not need to add it explicitly.